pokemon-showdown

```diff
@@ -2,14 +2,16 @@
 'use strict';
 
 const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
 
 // Make sure we're Node 6+
 
 try {
 	eval('{ let [a] = [1]; }');
 } catch (e) {
 	console.log("We require Node.js v6 or later; you're using " + process.version);
-	process.exit();
+	process.exit(0);
```

Comment:
It looks like it uses [C error codes](http://www.virtsync.com/c-error-codes-include-errno) judging from the values for `process.exitCode` used in its [unit tests](https://github.com/nodejs/node/blob/8b76c3e60c7b5c274c757257580a2c0faae69097/test/parallel/test-process-exit-code.js)